src/server/database/database_provider.py

```diff
@@ -29,29 +28,19 @@ class DatabaseProvider:
     """
 
     # predeclaration on the class level
-    connection: sqlite3.Connection = sqlite3.Connection(":memory:")
-    database_source: str = ""
+    connection: psycopg.Connection = None  # type: ignore
+    db_name: str = "doeqyyge"
+    db_user: str = "doeqyyge"
+    db_password: str = "sc_3pV9L4dezgJzWWh5wZMtJBhCiBvcS"
+    db_hostname: str = "snuffleupagus.db.elephantsql.com"
+    db_connection_timeout: int = 30
```

Comment:
Please use environment variables for all above.
Now that you exposed those on public repo, I would also advise to change password.

Reply:
Done